starfish/util/synthesize.py

```diff
@@ -0,0 +1,202 @@
+import random
+from itertools import product
+from typing import Tuple
+
+import numpy as np
+from numpy import zeros, array
+from numpy.random import rand, normal, poisson
+from pandas import DataFrame, concat
+from skimage.exposure import rescale_intensity
+from skimage.filters import gaussian
+from slicedimage import Tile, TileSet
+from starfish.constants import Indices, Coordinates
```

Comment:
needs NL before this line.

Reply:
Yea, sorry. Pycharm doing weird things with optimize imports on one of my laptops.